sympy/solvers/solvers.py

```diff
@@ -2296,8 +2296,13 @@ def solve_linear_system(system, *symbols, **flags):
     {}
 
     """
+    do_simplify = flags.get('simplify', False)
+
     assert system.shape[1] == len(symbols) + 1
 
+    if(do_simplify):
+        system = system.applyfunc(simplify)
```

Comment:
Brackets are not needed around the `if` condition in Python

Reply:
Sure, I'll fix that. That was by mistake.